draft-packages/illustration/KaizenDraft/Illustration/illustration.spec.tsx

```diff
@@ -4,7 +4,9 @@ import * as SpotIllustrations from "./Spot"
 import * as SceneIllustrations from "./Scene"
 
 afterEach(cleanup)
-
+beforeEach(() => {
+  jest.spyOn(global.console, "warn").mockImplementation(jest.fn())
```

Comment:
without any mock restoration, would this end up unintentionally silencing other warnings?

Reply:
I had lazily assumed cleanup would do that but that is apparently not the case. So probably, thanks.

Reply:
Fixed (I think!? spy confuses me) in the other PR and rebased.